packages/plugins/graphql/server/src/services/constants.ts

```diff
@@ -1,5 +1,7 @@
 const PAGINATION_TYPE_NAME = 'Pagination';
+const DELETE_MUTATION_RESPONSE_TYPE_NAME = 'DeleteMutationResponse';
 const PUBLICATION_STATE_TYPE_NAME = 'PublicationState';
+const PUBLICATION_STATUS_TYPE_NAME = 'PublicationStatus';
```

Comment:
Do we still need PUBLICATION_STATE_TYPE_NAME?

Reply:
nope nice catch